src/results/existing_chiller.jl

```diff
@@ -39,7 +39,7 @@ function add_existing_chiller_results(m::JuMP.AbstractModel, p::REoptInputs, d::
     r = Dict{String, Any}()
 
 	@expression(m, ELECCHLtoTES[ts in p.time_steps],
-		sum(m[:dvProductionToStorage][b,"ExistingChiller",ts] for b in p.s.storage.types.cold, t in p.techs.cooling)
```

Comment:
@zolanaj This was the issue. Just a results thing. When both ExistingChiller and AbsorptionChiller are included in techs.cooling, this ELECCHLtoTES expression was double counted.

Reply:
@Bill-Becker that'll definitely do it, thanks for catching this!

Reply:
@Bill-Becker is this still a draft, or can I review more thoroughly?

Reply:
I've made the changes I wanted to make, and then I wasn't sure if we wanted to just leave this PR as a placeholder for adding the default cost/cop data and logic. If you want to address that later, this is ready to review.

Reply:
@Bill-Becker thanks, I'll review today in that case, so that we can get the bug fix out (the lack of defaults and cost curves make it a simpler cost model for the Absorption chiller, but it's functional, and we have the issue in place to make sure that feature is added. I'll assign the issue to myself, mark this ready for review, and follow up later today.